File: Java/DesignPatterns/src/main/java/designPatterns/structural/facade/SubClass.java
package designPatterns.structural.facade;

public class SubClass {
  public static class A {
    public String doSomethingA() {
      return "SubClassA";
    }
  }

  public static class B {
    public Integer doSomethingB() {
      return 100;
    }
  }

  public static class C {
    public Boolean doSomethingC() {
      return true;
    }
  }
}

File: Kotlin/DesignPatterns/src/main/kotlin/structural/facade/SubClass.kt
package structural.facade

sealed class SubClass {
  class A {
    fun doSomethingA(): String {
      return "SubClassA"
    }
  }

  class B {
    fun doSomethingB(): Int {
      return 100
    }
  }

  class C {
    fun doSomethingC(): Boolean {
      return true
    }
  }
}
